website/docs/r/database_vulnerability_assessment_rule_baseline.html.markdown

```diff
@@ -0,0 +1,159 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_database_vulnerability_assessment_rule_baseline"
+sidebar_current: "docs-azurerm-resource-database-vulnerability-assessment-rule-baseline-x"
+description: |-
+  Manages a Database Vulnerability Assessment Rule Baseline.
+
+---
+
+# azurerm_database_vulnerability_assessment_rule_baseline
+
+Manages a Database Vulnerability Assessment Rule Baseline.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
```

Comment:
Could we use example instead of test for resource names?
```suggestion
resource "azurerm_resource_group" "example" {
```

Reply:
done